lib/console/devices/devices.ex

```diff
@@ -172,4 +172,10 @@ defmodule Console.Devices do
     devices = from(d in Device, where: d.organization_id == ^organization.id) |> Repo.all()
     length(devices)
   end
+
+  def update_in_xor_filter(device_ids, value) do
+    from(d in Device, where: d.id in ^device_ids) |> Repo.update_all(set: [in_xor_filter: value])
```

Comment:
Given how we use this function in the controller file, it doesnt seem necessary to do the next line. We can just return the result of Repo.update_all

Reply:
you is right